packages/babel-core/test/parse.js

```diff
@@ -10,16 +9,20 @@ function fixture(...args) {
 describe("parse", function() {
   it("should parse using configuration from .babelrc when a filename is provided", function() {
     const input = fs.readFileSync(fixture("input.js"), "utf8");
-    const output = fs.readFileSync(fixture("output.json"), "utf8");
-    assert(
-      parse(input, { filename: fixture("input.js"), cwd: fixture() }),
-      output,
-    );
+    const output = require(fixture("output"));
+
+    const result = parse(input, {
+      filename: fixture("input.js"),
+      cwd: fixture(),
+    });
+    expect(JSON.parse(JSON.stringify(result))).toEqual(output);
```

Comment:
Why is `JSON.parse(JSON.stringify())` needed?

Reply:
Because the returned AST returns classes, whereas the json file just has objects.

Reply:
@danez had suggested [it](https://github.com/babel/babel/pull/7513#issuecomment-371194571) to fix the errors described [here](https://gist.github.com/devenbansod/2d70d53e9d920ad77462a867c6937c83).